lib/config/deployment-config.ts

```diff
@@ -0,0 +1,113 @@
+import { getSecret } from "../utils/sm";
+
+interface InjectedConfigOptions {
+  project: string;
+  stage: string;
+  region?: string;
+}
+
+type InjectedConfigProperties = {
+  bootstrapUsersPasswordArn: string;
+  oktaMetadataUrl: string;
+  brokerString: string;
+  iamPath: string;
+  iamPermissionsBoundary: string;
+  vpcName: string;
+};
+
+export type DeploymentConfigProperties = InjectedConfigProperties & {
+  isDev: boolean;
+  project: string;
+  sharedOpenSearchDomainArn: string;
+  sharedOpenSearchDomainEndpoint: string;
```

Comment:
I think these openSearch ones are leftover from Mako

Reply:
Thank you